k4FWCore/scripts/k4run

```diff
@@ -155,6 +155,12 @@ def main():
         help="Print all the configurable components available in the framework and exit",
     )
     parser.add_argument("--gdb", action="store_true", help="Attach gdb debugger")
+    parser.add_argument(
+        "--disableROOTBatchMode",
```

Comment:
The rest of the options use dashes, maybe this could be changed to something like `--enable-root-interactive`? Or simply `--root-interactive`. I think the option in positive is easier to find in case someone wants to look at some plots.